.dev/edge-native-base/src/oauth2_proxy/app.py

```diff
@@ -57,8 +65,9 @@ def oauth_status():
             A 202 status if the user has an auth session
 
         """
-        if NATIVE_APP_MODE == "container" or NATIVE_APP_MODE == "dev":
+        if not HAVE_JUPYTERHUB:
             return "OK", 202
+        ping_server()
```

Comment:
This should be moved down into `if token` to prevent reporting of activity due to unauthorized access. (Unauthorized requests should not prevent a server from shutting down due to timeout)